### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by saundefined**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [x] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by ulivz** [comment]
@saundefined Before I merge it, could you keep the regular maintenance and updates? Thank you.

**Comment by saundefined** [comment]
@ulivz sure

**Comment by saundefined** [comment]
@ulivz May I add custom YAML tag `en: original file sha-1 hash` in translated files to track changes?
eg, like that: https://github.com/ru-docs/expo/blob/master/docs/index.md

**Comment by starkovsky** [comment]
"Google translate" too, as [PR 1663](https://github.com/vuejs/vuepress/pull/1663)

@ulivz Do not approved this PR please.

**Comment by ulivz** [comment]
@starkovsky Sorry but I'm not familiar with Russian,  are #1643 and #1644 both translation from google?

File: docs/ru/config/README.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+      md.use(require('markdown-it-xxx'))
+    }
+  }
+}
+```
+
+## Сборка Pipeline
+
+:::tip Настройка CSS препроцессоров
+VuePress поставляется со встроенным webpack для перечисленных ниже препроцессоров CSS. Для получения дополнительной информации об установке этих или препроцессоров без встроенной поддержки смотрите [Использование препроцессоров](../guide/using-vue.md#испоnьзование-препроцессоров) для получения дополнительной информации.
```
Comment (iNerV) [inline docs/ru/config/README.md]: `... этих или ...`
miss smth word/s

File: docs/ru/config/README.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+:::
+
+### postcss
+
+- Тип: `Object`
+- По умолчанию: `{ plugins: [require('autoprefixer')] }`
+
+Параметры для [postcss-loader](https://github.com/postcss/postcss-loader). Обратите внимание, что при указании этого значения автоматический префикс будет перезаписан, и вам нужно будет включить его самостоятельно.
```
Comment (iNerV) [inline docs/ru/config/README.md]: again google translator, autoprefixer proper name and not translated.

**Comment by starkovsky** [comment]
@ulivz Yes, PR #1644 and PR #1663.

File: docs/ru/config/README.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+```
+
+## Совместимость браузеров
+
+### evergreen
+
+- Тип: `boolean`
+- По умолчанию: `false`
+
+Установите значение `true`, если вы ориентируетесь только на вечнозеленые браузеры. Это отключит перенос ES5 и полизаполнения для IE, что приведет к более быстрой сборке и уменьшению размера файлов.
```
Comment (iNerV) [inline docs/ru/config/README.md]: again google translator, polyfill does not translate to russian

File: docs/ru/config/README.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+- По умолчанию: `undefined`
+
+Укажите языки для поддержки i18n. Подробнее смотрите в руководстве по [Интернационализации](../guide/i18n.md).
+
+### shouldPrefetch
+
+- Тип: `Function`
+- По умолчанию: `() => true`
+
+Функция для управления тем, какие файлы должны генерировать подсказки ресурса `<link rel="preload">`. Смотрите [ShouldPrefetch](https://ssr.vuejs.org/api/#shouldprefetch).
```
Comment (iNerV) [inline docs/ru/config/README.md]: what you mean `подсказки ресурса`?

**Comment by NataliaTepluhina** [comment]
@ulivz I would rather agree with @starkovsky, this translation quality is not that good and it's hard to read in Russian.

**Comment by sobolevn** [comment]
So, I guess this can be closed, because quality is very low. I (as a Russian) struggle to read this. 

Feel free to reopen if you want to improve the quality of the translation. Or create a new PR with the adapted version.

**Pull request closed without merging**